pkg/authorization/bootstrap/policy.go

```diff
@@ -46,6 +46,21 @@ const (
 	SystemKcpWorkspaceAccessGroup = "system:kcp:workspace:access"
 )
 
+const (
+	// SystemMastersGroup is the group inherited from k8s codebase - all powerful, all knowing!
+	// Users should not be added to this group.
+	SystemMastersGroup = "system:masters"
+)
+
+const (
+	// SystemKcpAdminUser is the user that has all permissions across all workspaces.
+	SystemKcpAdminUser = "kcp-admin"
+	// SystemShardAdmin is the user that has all permissions across all workspaces.
+	SystemShardAdmin = "shard-admin"
+	// SystemServiceAccountDefaultRest is the default service account for fake rest client.
+	SystemServiceAccountDefaultRest = "system:serviceaccount:default:kcp-rest"
```

Comment:
how is this special?

Reply:
with groups, not anymore too. But in general if we use this fake account in flows, wanted to make sure users cant create one. For edge cases we can't yet see.

Reply:
Would try to make all user name irrelavant.

Reply:
Just left it as constant. It irrelavant now.

Reply:
Where do we need it?